src/App.tsx

```diff
@@ -1,17 +1,98 @@
-import React from 'react';
-import './App.css';
-
+import React, { ChangeEvent, useState } from 'react';
 import users from './api/users';
+import todos from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { PreparedTodo } from './react-app-env';
+
+const preparedTodos: PreparedTodo[] = todos.map((todo) => {
+  const user = users.find(person => person.id === todo.userId) || undefined;
+
+  return (
+    {
+      ...todo,
+      user,
+    }
+  );
+});
```

Comment:
fine, but could be shorter
```suggestion
const preparedTodos: PreparedTodo[] = todos.map((todo) => (
    {
      ...todo,
      user: users.find(person => person.id === todo.userId),
    }
  );
);
```